packages/wrangler/src/experimental-assets.ts

```diff
@@ -149,18 +150,14 @@ export const syncExperimentalAssets = async (
 						body: payload.map((x) => JSON.stringify(x)).join("\n"),
 					}
 				);
+				assetUploadCount += payload.length;
 				logger.info(
-					`Uploaded bucket ${bucketIndex + 1}/${initializeAssetsResponse.buckets.length}`
+					`Uploaded ${assetUploadCount} of ${numberFilesToUpload} assets`
 				);
 				return res;
 			} catch (e) {
 				if (attempts < MAX_UPLOAD_ATTEMPTS) {
-					logger.info(
-						chalk.dim(
-							`Bucket ${bucketIndex + 1}/${initializeAssetsResponse.buckets.length} upload failed. Retrying...\n`,
-							e
-						)
-					);
+					logger.info(chalk.dim(`Asset upload failed. Retrying...\n`, e));
```

Comment:
we sure prev copy is not more useful for users?

Reply:
Eh I like these changes - I don't think buckets are really helping anyone meaningfully track any problem down. Plus they were a bit noisy because sometimes they were printing out of order, and I think sometimes the buckets can be empty.

No. of assets successfully uploaded seems more useful for users.

Reply:
Confirmed, we're going with the messaging I have here in this PR.